libraries/adaptive-expressions/src/builtinFunctions/isDateRange.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TimexProperty } from '@microsoft/recognizers-text-data-types-timex-expression';
+
+import { Expression } from '../expression';
+import { ExpressionEvaluator, ValueWithError } from '../expressionEvaluator';
+import { ExpressionType } from '../expressionType';
+import { FunctionUtils } from '../functionUtils';
+import { MemoryInterface } from '../memory/memoryInterface';
+import { Options } from '../options';
+import { ReturnType } from '../returnType';
+
+/**
+ * Return true if a given TimexProperty or Timex expression refers to a valid date range.

```

Comment:
>TimexProperty or Timex expression [](start = 26, length = 33)

All of these time related functions could use some common documentation around what these are.

Reply:
change to  "`TimexProperty` or Timex string". Is that ok?